gems/pending/appliance_console/database_replication_standby.rb

```diff
@@ -0,0 +1,82 @@
+require 'appliance_console/logging'
+require 'appliance_console/prompts'
+require 'appliance_console/database_configuration'
+require 'appliance_console/database_replication'
+
+module ApplianceConsole
+  class DatabaseReplicationStandby < DatabaseReplication
+    include ApplianceConsole::Logging
+
+    REGISTER_CMD = 'repmgr standby register'.freeze
+
+    attr_accessor :appliance_pg_data, :standby_host
```

Comment:
I think this should probably be `primary_host` rather than `standby_host` and then the host given during `ask_for_database_credentials` will have the same meaning as it does everywhere else.